data/v1/messages.json

```diff
@@ -42,7 +42,7 @@
                 },
                 "signatures": null,
                 "view_number": 0,
-                "vote_commitment": "COMMIT~tzMwZg2x5QJTx7Di9eEvNC0P8CZ_iNX7LJ01DHA9XEdO"
+                "vote_commitment": "COMMIT~f11eqw2KFTy-i-EaXUfEC1v0CPo8gPcd2My10Y5zSNok"
```

Comment:
Why does this change, and it is intentional?

Reply:
This is intentional! We missed a HotShot PR when cherry-picking into `release-gambit` which had a vote commitment fix, but we did not catch this until after we deployed `release-gambit` for cappuccino. Since we can't change what we've deployed, we reverted that change on HotShot `main`.

Note that the vote commitment here matches what's required to get the sequencer test to pass: https://github.com/EspressoSystems/espresso-sequencer/pull/2111

Reply:
@ss-es thanks! This would be good information to add to the PR or commit message.